src/main/java/easytests/api/v1/controllers/QuizzesController.java

```diff
@@ -63,4 +63,26 @@
     /**
      * show(quizId)
      */
+
+    @GetMapping("/{quizId}")
+    public Quiz show(@PathVariable Integer quizId) throws NotFoundException, ForbiddenException {
+        final QuizModelInterface quizModel = this.getQuizModel(
```

Comment:
nope, you shold use forAuth to fetch Quiz, that will be good for hasAcces mathod call.
ForAuth should be implemented in `getQuizModel(Integer id)` if there is no such method yet.